doc/developer/overview.md

```diff
@@ -108,7 +107,7 @@ They can be viewed as indexes of data that store a history of their indexed data
 Arrangements are how we share data across dataflows; many dataflows conclude by building an arrangement out of a collection of interest, and many dataflows start by importing the arrangements of collections of interest.
 
 To reduce memory use, arrangements allow users to enable "logical compaction" which removes the ability to access the data at distant historical moments (those not beyond the "logical compaction frontier").
-The more recent this logical compaction frontier the more compaction is allowed.
+The more recent this logical compaction frontier the more compaction is allowed and the less memory is requi.
```

Comment:
Word truncated here . Do those changes belong to this PR?

Reply:
These are doc changes that I did as part of the onboarding. I will move them in a separate PR.